src/rgw/rgw_admin.cc

```diff
@@ -303,6 +305,11 @@ void usage()
   cout << "                             object deletions by not involving GC\n";
   cout << "   --inconsistent-index      when specified with bucket deletion and bypass-gc set to true,\n";
   cout << "                             ignores bucket index consistency\n";
+  cout << "   --min-rewrite-size        specify the min object size condition for bucket rewrite (default 4M)\n";
+  cout << "   --max-rewrite-size        specify the max object size condition for bucket rewrite (default ULLONG_MAX)\n";
+  cout << "   --min-rewrite-stripe-size specify the min stripe size condition for object rewrite,\n";
+  cout << "                             default value is set to 0, in that case the specified object\n";
+  cout << "                             will always be rewritten for restriping\n";
```

Comment:
Usage is a shorter version of the actual description in the doc. So I think this can be shortened as 
`specify the min stripe size condition for object rewrite (default 0)`. What do you say ?

Reply:
OK, good idea!